src/like-service/composer.json

```
@@ -8,12 +8,16 @@
     ],
     "license": "MIT",
     "require": {
-        "php": "^7.3",
+        "php": "^8.0",
         "fideloper/proxy": "^4.2",
         "fruitcake/laravel-cors": "^2.0",
         "guzzlehttp/guzzle": "7.4.3",
-        "laravel/framework": "8.2",
-        "laravel/tinker": "^2.0"
+        "laravel/framework": "8.22",
```

[ISSUE] I think this needs to be
```suggestion
        "laravel/framework": "8.22.0",
```
otherwise I would assume that composer just picks any of the available versions